device/futhark/include/traccc/futhark/cca.hpp

```diff
@@ -0,0 +1,29 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include "traccc/edm/cell.hpp"
+#include "traccc/edm/measurement.hpp"
+#include "traccc/utils/algorithm.hpp"
+#include "vecmem/memory/memory_resource.hpp"
+
+struct futhark_context_config;
+struct futhark_context;
```

Comment:
There are not actually needed here, right? :confused:

Reply:
Indeed not, removed.